src/Avalonia.Controls/VirtualizingStackPanel.cs

```diff
@@ -51,7 +52,25 @@ public class VirtualizingStackPanel : VirtualizingPanel, IScrollSnapPointsInfo
             RoutedEvent.Register<VirtualizingStackPanel, RoutedEventArgs>(
                 nameof(VerticalSnapPointsChanged),
                 RoutingStrategies.Bubble);
+        /// <summary>
+        /// Defines the <see cref="BufferFactor"/> property.
+        /// </summary>
+        public static readonly StyledProperty<double> BufferFactorProperty =
```

Comment:
One thing to consider: WinUI/UWP has a similar concept as this in `ItemsRepeater` and the properties which control this behavior there are called `HorizontalCacheLength` and [`VerticalCacheLength`](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.itemsrepeater.verticalcachelength?view=windows-app-sdk-1.7).

As `VirtualizingStackPanel` can only virtualize on one axis, we don't need two properties but we should consider whether we want to adopt similar naming? We'll do an API review on this at the end anyway.

Reply:
I do not have a preference on the naming.
I thing you guys are the owners and should therefore tell me how to name stuff.

That being said: I wanted to use a relative value, therefore called "factor"
`Length` sounds like a fixed pixel length that is used for caching. 

So, we could call it `VirtualizationFactor` or `CachingFactor`?

Reply:
Not sure just yet - we probably need to do a quick API review with the core team.

Reply:
please do :-)